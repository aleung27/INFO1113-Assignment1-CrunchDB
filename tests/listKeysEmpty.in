LIST KEYS
SET a 1
GET a
POP a
LIST KEYS
SET b 9
LIST KEYS
