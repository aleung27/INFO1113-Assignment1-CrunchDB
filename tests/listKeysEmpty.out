> no keys

> ok

> [1]

> 1

> a

> ok

> b
a

